src/modality-specific-files/magnetic-resonance-spectroscopy.md

```diff
@@ -0,0 +1,191 @@
+while the other stores the sum of the transients.
+Bruker stores the sequence name, voxel position, voxel orientation, and other metadata in a separate plaintext file.
+These files are considered source data and, if present, MUST be stored in the
+[`/sourcedata`](../common-principles.md#source-vs-raw-vs-derived-data) directory.
+
+Due to the diversity in manufacturers' MRS data file formats, source data MUST be converted into the
+[NIfTI-MRS format](https://doi.org/10.1002/mrm.29418) (`*.nii[.gz]`).
+This format is based on the NIfTI framework and is designed to accommodate the nuances of raw MRS data.
+All necessary information to parse this `*.nii[.gz]` file (for example, spectrometer frequency, echo time,
+repetition time, and so on) are stored in a JSON header extension.
+Conversion of proprietary MRS file formats to NIfTI-MRS and extraction of some (but not all) BIDS-compliant metadata can be performed
+using [spec2nii](https://github.com/wtclarke/spec2nii).
+Note that the "rawness" of data stored in the NIfTI-MRS file will depend on the format of the source data.
+It is RECOMMENDED that users export their source data from the scanner in an appropriately raw format prior to conversion.
+
+For MRSI data, "raw" signifies spatially reconstructed data (that is, data in image space rather than (*k*,*t*)-space),
+given the complexity and diversity of sampling approaches.
+Note that NIfTI-MRS is not designed to store data that has not been spatially reconstructed.
+
+### Single-voxel spectroscopy and MRS imaging
+
+| **Name**                                 | **`suffix`** | **Description**                                                                                                                                                                       |
+| ---------------------------------------- | ------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Single-voxel spectroscopy                | svs          | MRS acquisitions where the detected MR signal is spatially localized to a single volume.                                                                                              |
+| Magnetic resonance spectroscopic imaging | mrsi         | MRS acquisitions where additional imaging gradients are used to detect the MR signal from 1, 2, or 3 spatial dimensions.                                                              |
+| Unlocalized spectroscopy                 | unloc        | MRS acquisitions run without localization. This includes signals detected using coil sensitivity only.
+|
+| Concentration or calibration reference   | ref          | A separate MRS acquisition acquired to detect a signal to serve as a concentration reference for absolute quantification or for preprocessing (for example, eddy-current correction). 
```

Comment:
I think `ref` is too general a term to want it to go bare. Would `refspec` or `specref` for reference spectroscopy or spectroscopy reference work? The comparison I'm thinking of here is "single-band reference" (`sbref`) to accompany multiband BOLD or DWI sequences.

In those cases, `sbref` generally matches `bold` or `dwi` images in all but suffix. Is that expected here, or would one `ref` be good for the entire session?

Reply:
How often is a single-band reference image acquired when MRI users perform multi-band MRI? Because acquiring a reference MRS acquisition is pretty much standard in MRS. I'm not sure if we should have to conform to MRI when it's far more niche to MRI than MRS.

Reply:
I've thought about this, and you're right: I suggest we use `specref`.

Reply:
Thinking back on this: `specref` could eventually (potentially) be confused with SPECT (assuming some group decides to BIDSify SPECT - which someone probably should...).

I guess `mrsref` would make more sense.

Reply:
Addressed in [248de47](https://github.com/bids-standard/bids-specification/pull/1377/commits/248de47ec238325a784df3ecfda2a3d053f3b8a1)